### ISSUE TITLE
Not able to get reasonable results

### ISSUE THREAD

**Issue by zengxianyu** [ZongyuGuo/Inpainting_FRRN#2]
Hi Zhongyu, 

Thank you for your code. I tried to process images using parameters ./InpaintingModel_gen.pth and called function ```model.process``` but it doesn't provide reasonable results. That's how I use this function 
```python
img = Image.open(os.path.join(config.image, name+".jpg")).convert("RGB")
mask = Image.open(path_to_hole_mask).convert("L")
img = np.array(img).transpose((2, 0, 1))[None, ...]
mask = np.array(mask)[None, None, ...]
img = torch.Tensor(img).float().cuda()
mask = (torch.Tensor(mask)>0).float().cuda()
outputs, gen_loss, dis_loss = inpaint_model.process(img, 1-mask)
outputs = -F.threshold(-outputs, -255.0, -255.0)
img_out = outputs[0].detach().permute(1, 2, 0).cpu().numpy()
 img_out = Image.fromarray(img_out.astype(np.uint8))
img_out.save(path_to_save)
```

But the results are too dark. Did I call this model right?

**Comment by zengxianyu** [comment]
the masks I provided have value 1 for missing pixels and 0 for valid pixels

**Comment by ZongyuGuo** [comment]
Input should be normalized to [0,1] instead of [0,255]. I have changed your code to

    img = Image.open('image.png').convert("RGB")
    mask = Image.open('mask.png').convert("L")
    img = np.array(img).transpose((2, 0, 1))[None, ...]
    mask = np.array(mask)[None, None, ...]
    img = torch.Tensor(img).float().cuda() / 255.0
    mask = (torch.Tensor(mask)>0).float().cuda()
    outputs, gen_loss, dis_loss = inpaint_model.process(img, 1-mask)
    img_out = outputs[0].detach().permute(1, 2, 0).cpu().numpy() * 255.0
    img_out = Image.fromarray(img_out.astype(np.uint8))
    img_out.save('./image_rec.png')

And it works well now.

**Comment by zengxianyu** [comment]
Yes it works, thank you very much
